### PR TITLE
Flight school ruby pumped to 2.4.0 and gem dependency issue resolved

### PR DESCRIPTION
**Pull request by thaniyarasu**
Flight school ruby pumped to 2.4.0 and gem dependency issue resolved

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@thaniyarasu Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/flight-school&pullRequestId=8)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=concourse/flight-school&pullRequestId=8) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@thaniyarasu Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/flight-school&pullRequestId=8)!

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 RSpec.configure do |config|
+  
+  config.filter_run_including foo: :bar
```
Comment (chendrix) [inline spec/spec_helper.rb]: What's this line for?

**Comment by thaniyarasu** [comment]
sorry, it was typo mistake. updated PR

**Comment by chendrix** [comment]
@thaniyarasu Have you verified that following the documentation on http://concourse.ci/flight-school.html still works? If it does, can you also make a PR to https://github.com/concourse/concourse/blob/master/docs/tutorials/flight-school.any?

**Comment by thaniyarasu** [comment]
@chendrix : sure, i will add a PR for that

**Comment by chendrix** [comment]
Thanks again for this @thaniyarasu

**Pull request merged**